plugins/modules/packaging/language/maven_artifact.py

```diff
@@ -95,6 +95,18 @@
             - If C(no), SSL certificates will not be validated. This should only be set to C(no) when no other option exists.
         type: bool
         default: 'yes'
+    client_cert:
+        description:
+            - PEM formatted certificate chain file to be used for SSL client authentication.
+            - This file can also include the key as well, and if the key is included, C(client_key) is not required.
+        type: path
+        version_added: '1.3.0'
+    client_key:
+        description:
+            - PEM formatted file that contains your private key to be used for SSL client authentication.
+            - If C(client_cert) contains both the certificate and key, this option is not required.
```

Comment:
```suggestion
            - If I(client_cert) contains both the certificate and key, this option is not required.
```

Reply:
I'm just curious what the change from C to I does.

Reply:
`I(...)` is used for option names, `C(...)` for option values (and others). See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation